lib/project_types/extension/models/specification_handlers/checkout_post_purchase.rb

```diff
@@ -6,12 +6,12 @@ module Models
     module SpecificationHandlers
       class CheckoutPostPurchase < Default
         PERMITTED_CONFIG_KEYS = [:metafields]
-        RENDERER_PACKAGE_NAME = "@shopify/post-purchase-ui-extensions"
+        RENDERER_PACKAGE_NAME = "@shopify/post-purchase-ui-extensionss"
```

Comment:
I think this was accidentally changed

Reply:
oh I did that intentionally to tophat an error case 😅